test/integration/test_good_templates.py

```diff
@@ -251,38 +148,12 @@ class TestQuickStartTemplates(BaseCliTestCase):
                     }
                 }
             ],
-            'exit_code': 6,
+            'exit_code': 2,
```

Comment:
**transforms into template that previously violated [`W3011`](https://github.com/aws-cloudformation/cfn-python-lint/blob/fe0af928a4ab26a6cedfd76564a26c9a4120b14f/src/cfnlint/rules/resources/BothUpdateReplacePolicyDeletionPolicyNeeded.py):**

[`cfn-lint --info test/fixtures/templates/good/transform_serverless_globals.yaml 2>&1 | grep -C 7 'DeletionPolicy'`](https://github.com/aws-cloudformation/cfn-python-lint/blob/fe0af928a4ab26a6cedfd76564a26c9a4120b14f/test/fixtures/templates/good/transform_serverless_globals.yaml)

```
      "myFunctionVersionee13cf2679": {
        "DeletionPolicy": "Retain",
        "Properties": {
          "FunctionName": {
            "Ref": "myFunction"
          }
        },
        "Type": "AWS::Lambda::Version"
      }
```

https://github.com/aws-cloudformation/cfn-python-lint/blob/fe0af928a4ab26a6cedfd76564a26c9a4120b14f/src/cfnlint/core.py#L60-L61